server/charts/oca-repository/templates/ingress.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "oca-repository.fullname" . }}-ingress
+  namespace: {{ .Values.namespace | default .Release.Namespace }}
+  labels:
+    {{- include "oca-repository.labels" . | nindent 4 }}
+  annotations:
+    {{- toYaml .Values.ingress.annotations | nindent 4 }}
+spec:
+  ingressClassName: {{ .Values.ingress.className }}
+  rules:
+    - host: oca-repository.marketplace.{{ .Values.ingress.hostBase }}
```

Comment:
you should use {{values.namespace}} instead of writing the marketplace like this as it should be updated if the name space updated